{ 'cebit_handshake': { 'head': [ { 'duration': 1.96,
                                   'name': u'cebit_right_down',
                                   'positions': [-0.3768, 0.3817],
                                   'starttime': 0.0}],
                       'left_arm': [],
                       'left_leg': [],
                       'right_arm': [ { 'duration': 1.96,
                                        'name': u'cebit_hand_shake_3',
                                        'positions': [ 1.6903,
                                                       -0.5028,
                                                       0.3599,
                                                       0.1258,
                                                       0.5999,
                                                       0.25,
                                                       -1.47],
                                        'starttime': 0.0}],
                       'right_leg': [],
                       'torso': []}}
